contributing/tools.md

```diff
@@ -0,0 +1,37 @@
+# Tools that may save you and the team some time
+
+## git-clang-format
+
+This tool allows you to run clang-format on your local changes before they're sent out for review.
+This will minimize the amount of style feedback you get from the team and saves everyone time.
+
+```bash
+# Check to see if you have clang-format command installed.
+# If not, get the latest release from https://github.com/material-foundation/clang-format/releases
```

Comment:
I believe the team should be using the internal version, yes? Should we install using our internal package manager?